src/pages/RepoPage/PullsTab/PullsTable/createPullsTableData.tsx

```diff
@@ -1,34 +1,67 @@
 import isArray from 'lodash/isArray'
 
 import { Pull } from 'services/pulls/usePulls'
+import { formatSizeToString } from 'shared/utils/bundleAnalysis'
 import TotalsNumber from 'ui/TotalsNumber'
 
-import Coverage from './Coverage'
+import Title from './Title'
 
-import Title from '../shared/Title'
+export const ErroredUpload = () => (
+  <p>
+    Upload: <span aria-label="Errored upload">&#x274C;</span>
+  </p>
+)
+export const PendingUpload = () => (
+  <p>
+    Upload: <span aria-label="Pending upload">&#x23F3;</span>
+  </p>
+)
 
 export const createPullsTableData = ({ pulls }: { pulls?: Array<Pull> }) => {
   if (!isArray(pulls)) {
     return []
   }
+
   return pulls.filter(Boolean).map((pull: Pull) => {
-    let patch, change
-    if (pull?.compareWithBase?.__typename === 'Comparison') {
-      patch = pull?.compareWithBase?.patchTotals?.percentCovered ?? 0
-      change = pull?.compareWithBase?.changeCoverage ?? 0
+    let patch = <p>-</p>
+    if (pull?.head?.coverageStatus === 'ERROR') {
```

Comment:
There an enum for these? 'ERROR', 'PENDING' and 'COMPLETED'

Reply:
Don't really need it, the `coverageStatus` type is a union of string literals `ERROR | PENDING | COMPLETED`, so it'll error and fail the build if you change the return type. Also using strings directly like this is overall better for compression and smaller bundle sizes

Reply:
> overall better for compression and smaller bundle sizes

If you have a source on this, would love to read more about the magnitude of gains here.

with respect to it erroring on the build by changing the return type, I'm sure there is a way to update the zod schema to account for this type on either end even with an enum.

In any case, just a thought that may improve readability for some

Reply:
Here's an example PR of swapping to regular strings from the object style enum I believe you're talking about: https://github.com/getsentry/sentry-javascript/pull/4280. 

A different concern with using the object syntax like we do in Gazebo (which I think we need to iterate on, and move away from) is that the field names themselves can not be minified. There are also issues with tree shaking that come up since we code split quite heavily so this also brings up issues with bundle sizes.

An alternative I'd be alright exploring would be something more along the lines of:

```ts
export const COMMIT_STATUS_COMPLETED = 'COMPLETED'
export const COMMIT_STATUS_ERROR = 'ERROR'
export const COMMIT_STATUS_PENDING = 'PENDING'

const CommitStatusSchema = z.union([
  z.literal(COMMIT_STATUS_COMPLETED),
  z.literal(COMMIT_STATUS_ERROR),
  z.literal(COMMIT_STATUS_PENDING),
])
```

Reply:
Very neat! 2% savings definitely adds up over time, thanks for sharing that.

I think having a shared schema like that solves my original concern which is the improved readability, and we won't need the object syntax either. I'm all for it